modules/ROOT/pages/streaming-strategies-reference.adoc

```diff
@@ -3,58 +3,66 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-== Repeatable in Memory Stream Reference
+You can use the default streaming strategy settings or configure then for your
```

Comment:
then > them ? or maybe "configure **one** for your needs"

Reply:
done